### DISCUSSION TITLE
How to use content script without vue UI

### DISCUSSION THREAD

**Discussion by realrecordzLab** [mubaidr/vite-vue3-chrome-extension-v3#123] [General]
Is possible to just use content script without the vue ui? I was using another boilerplate where just the index of the content script were used, I want to do the same with this nice boilerplate.

**Comment by mubaidr** [comment]
Yes, why not! Content ui frame is injected in this file: https://github.com/mubaidr/vite-vue3-browser-extension-v3/blob/master/src/content-script/index.ts#L11

Just empty this file and use as regular module to perform any custom functionality.
